filters-share/specific_ELEMHIDE.txt

```diff
@@ -22,7 +22,9 @@ dcinside.com##div[class$="banner"]
 toomics.com,nongmin.com,ilbe.com##div[class$="-banner"]
 ! ##div[class^="banner"]
 m.edaily.co.kr,ichannela.com,m.news1.kr##div[class^="banner"]
-! 
+!
+m.10000recipe.com##.view_sp_banner
+m.10000recipe.com###_fixedBannerAppInstall
```

Comment:
List-KR does not block the app notification.
https://github.com/AdguardTeam/AdguardFilters/pull/118593

Reply:
ok